cbrkit/retrieval.py

```diff
@@ -99,6 +115,28 @@ def build(
     similarity_func: AnySimFunc[KeyType, ValueType, SimType],
     limit: int | None = None,
 ) -> RetrieveFunc[KeyType, ValueType, SimType]:
+    """Based on the similarity function this function creates a retriever function.
+
+    Args:
+        similarity_func: Similarity function to compute the similarity between cases.
+        limit: Retriever function will return the top limit cases.
+
+    Returns:
+        Returns the retriever function.
+
+    Examples:
+        >>> retriever = cbrkit.retrieval.build(
```

Comment:
Since all examples are assumed to be run in the Python interpreter, this yields a syntax error. If wanting to use multiple lines, I think you need to use `...` instead of `>>>` at the beginning of the line. But that will be documented in the doctest documentation.